src/constants/constants.js

```diff
@@ -0,0 +1,65 @@
+export const politicalPositions = [Object({
+  label: 'President',
+  value: 'president',
+}), Object({
+  label: 'Vice Precident',
+  value: 'vice',
+}), Object({
+  label: 'Governor',
+  value: 'governor',
+}), Object({
+  label: 'Deputy Governor',
+  value: 'deputy',
+}), Object({
+  label: 'Senator',
+  value: 'senator',
+}), Object({
+  label: 'Minister',
+  value: 'minister',
+}), Object({
+  label: 'Commissioner',
+  value: 'commissioner',
+}), Object({
+  label: 'Local Government Chairman',
+  value: 'local',
+})];
+
+export const states = [
```

Comment:
I feel like we could have this as its own json data file

Reply:
Done @chinezenwosu